### ISSUE TITLE
A bit of feedback

### ISSUE THREAD

**Issue by tin2tin** [blenderskool/blender-depot-importer#3]
Great and much needed project.

I've been collecting a number of Text Editor add-ons here: https://blenderartists.org/t/how-would-you-show-some-love-for-the-blender-text-editor/1163857/ I thought it would be great to collect all of the add-ons in one bundle for people use, but searching for "Text Editor" a lot of add-ons not compatible with 2.80 shows up. 

But a lot of the add-ons do not have their own repository, but are a collection of add-ons in a repository, and therefore do not show up. So how can those be boxed too? And some of them needs to have additional modules installed with pip. Can anything be done about those? 

If blender-depot is using the info from the add-on files I guess the categories like Animation, Text Editor, Grease Pencil and version number could be used as keywords?
![image](https://user-images.githubusercontent.com/1322593/68390706-26f36a00-0166-11ea-8458-3140eca0e3bf.png)
And maybe the version number could be useful to extract too?

**Comment by blenderskool** [comment]
Thanks for the feedback. Right now, Blender Depot works only with add-ons with dedicated repositories on GitHub. **I have noticed that most of the add-ons don't have a repository**.

One of the ways to fix this would be to **allow user-submitted add-ons** in the platform, for example, you would be able to add your collection of text editor add-ons. The problem comes in how we can bundle it as a 'box' and distribute it. In some cases, it may not be a good idea to redistribute the add-on without the original author's permission.
In any case, user-submitted add-ons(even without bundling) would be helpful on the platform 😄

**Comment by tin2tin** [comment]
If you can import and use the info section of the add-ons, you could make bundles with ex. all 2.80 compatible add-ons. That would be really useful imo. and save some time from going back and forth to create your box selection. I mean these: 
```
bl_info = {
    "name": "New Object",
    "author": "Your Name Here",
    "version": (1, 0),
    "blender": (2, 80, 0),
    "location": "View3D > Add > Mesh > New Object",
    "description": "Adds a new Mesh Object",
    "warning": "",
    "wiki_url": "",
    "category": "Add Mesh",
}
```

**Comment by blenderskool** [comment]
@tin2tin Yes, the data in bl_info would be really useful to better organize the add-ons on Blender Depot. It is used by the importer add-on to install only compatible add-ons in the Blender version being used.
